README.md

```diff
@@ -101,7 +101,34 @@ would in a Django shell.
 
 Unlike our other web applications this project is not using tox. To run tests locally, run this command instead:
 
-    docker-compose run --rm web pytest
+    ### PYTHON TESTS/LINTING
+    # Run Python tests
+    
+    # if you do not already have a running container, you should replace exec with run --rm
```

Comment:
I'd recommend highlighting the portions like `you should replace "exec" with "run --rm" in the command`. Also, I'd recommend making it a heading like so:
### ...you should replace "exec" with "run --rm" in the command...

Reply:
its updated.